### DISCUSSION TITLE
Removing the Authentication token from the attribute widget

### DISCUSSION THREAD

**Discussion by lakhoune** [rwth-acis/Social-Bot-Framework#64] [Ideas]
Currently, unauthorized users could extract authentication tokens of bots if that bot was stored using the store function. 
To prevent this, we could implement it such that the frontend will prompt users to provide the authentication token every time they want to resubmit the bot or delete the bot.

**Comment by Aran30** [comment]
https://github.com/rwth-acis/Social-Bot-Framework/issues/13
